test/minter-suite-minters/MinterMerkle/MinterMerkleV1.test.ts

```diff
@@ -197,6 +211,127 @@ describe("MinterMerkleV1", async function () {
     });
   });
 
+  describe("payment splitting", async function () {
+    beforeEach(async function () {
+      this.userMerkleProofZero = this.merkleTreeZero.getHexProof(
+        hashAddress(this.accounts.user.address)
+      );
+      this.userMerkleProofOne = this.merkleTreeOne.getHexProof(
+        hashAddress(this.accounts.user.address)
+      );
+      this.additionalMerkleProofTwo = this.merkleTreeTwo.getHexProof(
+        hashAddress(this.accounts.additional.address)
+      );
+      this.deadReceiver = await deployAndGet.call(this, "DeadReceiverMock", []);
+    });
+
+    it("requires successful payment to platform", async function () {
+      // update platform address to a contract that reverts on receive
+      await this.genArt721Core
+        .connect(this.accounts.deployer)
+        .updateArtblocksPrimarySalesAddress(this.deadReceiver.address);
+      // expect revert when trying to purchase
+      await expectRevert(
+        this.minter
+          .connect(this.accounts.user)
+          ["purchase(uint256,bytes32[])"](
+            this.projectZero,
+            this.userMerkleProofZero,
+            {
+              value: this.pricePerTokenInWei,
+            }
+          ),
+        "Art Blocks payment failed"
+      );
+    });
+
+    it("requires successful payment to artist", async function () {
+      // update artist address to a contract that reverts on receive
+      await this.genArt721Core
+        .connect(this.accounts.deployer)
+        .updateProjectArtistAddress(
+          this.projectZero,
+          this.deadReceiver.address
+        );
+      // expect revert when trying to purchase
+      await expectRevert(
+        this.minter
+          .connect(this.accounts.user)
+          ["purchase(uint256,bytes32[])"](
+            this.projectZero,
+            this.userMerkleProofZero,
+            {
+              value: this.pricePerTokenInWei,
+            }
+          ),
+        "Artist payment failed"
+      );
+    });
+
+    it("requires successful payment to artist additional payee", async function () {
+      // update artist additional payee to a contract that reverts on receive
+      const proposedAddressesAndSplits = [
+        this.projectZero,
+        this.accounts.artist.address,
+        this.deadReceiver.address,
+        50,
+        this.accounts.additional2.address,
+        50,
```

Comment:
Nit: these two constant values could be helpful to add smol dev comments for.

Reply:
updated 🙏